frontend/src/graphql/events/queries.ts

```diff
@@ -0,0 +1,265 @@
+import { gql } from "@apollo/client";
+
+export const GET_EVENTS = gql`
+  query AllEvents($organization: String, $category: String, $startTime: DateTime, $endTime: DateTime) {
+    allEvents(organization: $organization, category: $category, startTime: $startTime, endTime: $endTime) {
+      id
+      title
+      startTime
+      endTime
+      location
+      description
+      organization {
+        name
+        color
+      }
+      category {
+        name
+      }
+      image
+      isAttendable
+      deadline
+      publisher {
+        id
+        username
+        email
+        firstName
+        lastName
+        dateJoined
+      }
+      price
+      shortDescription
+      signupOpenDate
+      userAttendance {
+        isSignedUp
+        isOnWaitingList
+        hasBoughtTicket
+      }
+      isFull
+      hasExtraInformation
+      allowedGradeYears
+    }
+  }
+`;
+
+export const GET_DEFAULT_EVENTS = gql`
+  query defaultEvents {
+    defaultEvents {
+      id
+      title
+      startTime
+      endTime
+      location
+      description
+      organization {
+        name
+        color
+      }
+      category {
+        name
+      }
+      image
+      isAttendable
+      deadline
+      publisher {
+        id
+        username
+        email
+        firstName
+        lastName
+        dateJoined
+      }
+      price
+      shortDescription
+      signupOpenDate
+      userAttendance {
+        isSignedUp
+        isOnWaitingList
+      }
+      isFull
+      hasExtraInformation
+      allowedGradeYears
+    }
+  }
+`;
+
+export const GET_EVENT = gql`
+  query Event($id: ID!) {
+    event(id: $id) {
+      id
+      title
+      startTime
+      endTime
+      location
+      description
+      organization {
+        id
+        name
+      }
+      category {
+        id
+        name
+      }
+      image
+      isAttendable
+      deadline
+      publisher {
+        id
+        username
+        email
+        firstName
+        lastName
+        dateJoined
+      }
+      availableSlots
+      price
+      shortDescription
+      signupOpenDate
+      userAttendance {
+        isSignedUp
+        isOnWaitingList
+        hasBoughtTicket
+        positionOnWaitinglist
```

Comment:
To get this across the finish line, move `positionOnWaitinglist` to `fragments.graphql` > `Event` in the same folder. We've moved away from the these files recently. Afterwards, run `yarn generate` in `/frontend` to get updated typescript types.

Afterwards, you can delete this entire file.

Reply:
yarn generate gets the followint error: 

PS C:\Users\hafst\indok-web\frontend> yarn generate
yarn run v1.22.19
$ graphql-codegen --config codegen.yml
✔ Parse Configuration
⚠ Generate outputs
  ❯ Generate to src/generated/graphql.tsx
    ✔ Load GraphQL schemas
    ✔ Load GraphQL documents
    ✖ GraphQL Document Validation failed with 23 errors;
      Error 0: GraphQLDocumentError: Cannot query field "positio…
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

I have no idea why this happens, however i have not dared to delete [frontend/src/graphql/events/queries.ts] so if that is a problem it explains the error.